src/main/java/com/minecolonies/coremod/util/TeleportToColony.java

```diff
@@ -0,0 +1,85 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.IColony;
+import com.minecolonies.coremod.commands.MinecoloniesCommand;
+import com.minecolonies.coremod.configuration.Configurations;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Helper class for teleporting players to home or a friendly colony.
+ */
+public final class TeleportToColony
+{
+    private static final String CANT_FIND_COLONY = "No Colony found for teleport, please define one.";
+    private static final String CANT_FIND_PLAYER = "No player found for teleport, please define one.";
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    public TeleportToColony(@NotNull final String... parents)
+    {
+
+    }
+
+    @NotNull
+    public static void ColonyTeleport(@NotNull MinecraftServer server, @NotNull ICommandSender sender, @NotNull String... args){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00100)